nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/TeamAutomationHandler.java

```diff
@@ -367,4 +386,11 @@ private TeamMemberSlackIdentity getTeamMemberSlackIdentity(
         AtomistMemberBase requestedBy;
     }
 
+    @Value
+    private class teamSlackChannelCreated {
+        private TeamCreated team;
```

Comment:
Rather use the AtomistTeam type - these AtomistXXX types are the ones we have started using and maintaining between Gluon and QM.
```suggestion
        private AtomistTeam team;
```
Will need to include the dependency on this one too I think.